framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenesStorage.java

```diff
@@ -361,7 +362,7 @@ public boolean addMethodDeclarationAnnotation(
 
     AMethod methodAnnos = getMethodAnnos(methodElt);
 
-    scenelib.annotations.Annotation sceneAnno =
+    org.checkerframework.afu.scenelib.annotations.Annotation sceneAnno =
```

Comment:
I don't think a fully-qualified name is necessary here, unless done for emphasis.